test/test_correlation_sampling.jl

```diff
@@ -128,7 +128,7 @@ end
     # Merge correlations and check if result equal to running calculation.
     sc_merged = merge_correlations([sc1, sc2])
     @test sc0.data ≈ sc_merged.data
-    @test sc0.variance ≈ sc_merged.variance
+    @test sc0.M[1] ≈ sc_merged.M[1]
```

Comment:
Should this hold for the whole `M` array?

Reply:
Yes. Fixed.